crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -1174,15 +1182,16 @@ impl TryFrom<StoredTransaction> for TransactionBlock {
             TransactionExpiration::Epoch(epoch_id) => Some(*epoch_id),
         };
 
-        let kind = TransactionBlockKind::from(sender_signed_data.transaction_data().kind());
+        //TODO add kind when PTBs are implemented
+        // let kind = TransactionBlockKind::from(sender_signed_data.transaction_data().kind());
```

Comment:
as an alternative to getting rid of all the TransactionBlockKind code, why don't we create some simple representation of PTBkind for now, could literally just be a string that says "PTB" kind. Not a fan of removing all the other kinds

Reply:
I'll look into it.

Reply:
Fixed